git-ftp

```diff
@@ -1792,9 +1792,23 @@ do
 				write_log "Disabling EPSV."
 			fi
 			;;
-		--remote-root)
-			REMOTE_ROOT="$2"
-			shift
+		--remote-root*)
+			case "$#,$1" in
+				*,*=*)
+					REMOTE_ROOT=$(expr "z$1" : 'z-[^=]*=\(.*\)')
+					;;
+				1,*)
+					print_error_and_die "Too few arguments for option --key." "$ERROR_MISSING_ARGUMENTS"
+					;;
+				*)
+					if ! echo "$2" | egrep -q '^-'; then
+						REMOTE_ROOT="$2"
+						shift
+					else
+						print_error_and_die "Too few arguments for option --key." "$ERROR_MISSING_ARGUMENTS"
```

Comment:
Same here 😆 

I think there is a copy & past error. It should be `option --remote-root`